### PR TITLE
Added clearer instructions to the "Create Many Records with model.create()" challenge

### LINKED ISSUE #36447: Broken Test For "MongoDB and Mongoose - Create Many Records with model.create()"
The following code produces a `Timeout has occurred` message in the freeCodeCamp console, yet no errors are shown on the Glitch console, and the data does get saved to the MongoDB Atlas.

```
const { Schema } = mongoose;

const personSchema = new Schema({
  name: String,
  age: Number,
  favoriteFoods: [String],
});

const Person = mongoose.model('Person', personSchema);

const createManyPeople = (done) => {
  console.log('Done:', done);
  const arrayOfPeople = [
    { name: 'Meeva', age: 30, favoriteFoods: ['Fettuccine Alfredo', 'Sushi', 'Quiche'] },
    { name: 'Leonardo', age: 44, favoriteFoods: ['Pasta', 'Cheeseburgers', 'French Fries'] },
  ];
  Person.create(arrayOfPeople, (err, data) => {
    if (err) return done[err];
    return done[null, data];
  });
};
```

Furthermore, the Glitch console reports the following when I have the console log logging `done` enabled and the freeCodeCamp `I've completed this challenge` button is pressed:

```
OPTIONS
POST
Done: [ { name: 'John', age: 24, favoriteFoods: [ 'pizza', 'salad' ] },
  { name: 'Mary', age: 21, favoriteFoods: [ 'onions', 'chicken' ] } ]
```

### PR DESCRIPTION
**Pull request by Anonyma**
The previous instructions were causing confusion:
https://www.freecodecamp.org/forum/t/mongodb-and-mongoose-create-many-records-with-model-create/208546
https://github.com/freeCodeCamp/freeCodeCamp/issues/36447

- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [ x] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [ x] My changes do not use shortened URLs or affiliate links.

Closes #36447

### CODE REVIEW

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-many-records-with-model.create.english.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ Sometimes you need to create many instances of your models, e.g. when seeding a
 
 ## Instructions
 <section id='instructions'>
-Create many people with <code>Model.create()</code>, using the function argument <code>arrayOfPeople</code>.
+Add a function expecting an <code>arrayOfPeople</code> argument which creates many people through <code>Model.create()</code>.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-many-records-with-model.create.english.md]:
```suggestion
Modify the <code>createManyPeople</code> function to create many people through <code>Model.create()</code> and the argument <code>arrayOfPeople</code>.
```
The function already exists. If a user creates a function with a different name, they would need to modify code in sections we've told them not to touch.

**Review by ojeytonwilliams — DISMISSED** [review]
Sorry for the extremely late review, this should have been looked at sooner.  Comments below:

**Review by moT01 — COMMENTED** [review]
I'm kind of leaning towards not adding these changes. If you open the [repl.it link](https://repl.it/repls/NanoHumbleBrackets#myApp.js) or [the boilerplate](https://github.com/freeCodeCamp/boilerplate-mongomongoose/blob/gomix/myApp.js#L87) it's fairly clear what to do. The function is there to be modified - and users in this area can figure that out I think. I guess the bigger reason I am leaning that way is because I would say we should modify the instructions in those two spots as well if we add this - which isn't worth it in my opinion.

**Comment by Sky020** [comment]
@moT01 I have **not** seen this confusion come up on the forum. So, I do not think there is a need to change the instructions. However, adding this would do no harm.

Personally, because the challenge occurs in the `myApp.js` file, I prefer the _extended_ instructions being in the `myApp.js` file. So, if any instructions are to change, I would have them be the ones in the boilerplate.

**Review by moT01 — DISMISSED** [review]
Sounds good to me @Sky020 - I say we add these changes to both places then. What do you think? I'll make a PR to the boilerplate.

If so, this LGTM 🎉

**Review by naomi-lgbt — DISMISSED** [review]
If we are on-board with adding the clarification, this PR fits that need and LGTM!

**Comment by Sky020** [comment]
@RandellDawson If you are happy with these changes, could you merge this? Personally, I am still on the edge with making changes to this lesson file, and would close #36447 with Tom's PR of the boilerplate.

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-many-records-with-model.create.english.md [hunk 2]
```diff
@@ -12,7 +12,8 @@ Sometimes you need to create many instances of your models, e.g. when seeding a
 
 ## Instructions
 <section id='instructions'>
-Create many people with <code>Model.create()</code>, using the function argument <code>arrayOfPeople</code>.
+Modify the <code>createManyPeople</code> function to create many people through <code>Model.create()</code> and the argument <code>arrayOfPeople</code>.
```
Comment (RandellDawson) [inline curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-many-records-with-model.create.english.md:15]:
```suggestion
Modify the <code>createManyPeople</code> function to create many people using <code>Model.create()</code> with the argument <code>arrayOfPeople</code>.
```

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-many-records-with-model.create.english.md [hunk 3]
```diff
@@ -12,7 +12,8 @@ Sometimes you need to create many instances of your models, e.g. when seeding a
...
+(You can reuse the model you instantiated in the previous exercise)
```
Comment (RandellDawson) [inline curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-many-records-with-model.create.english.md:16]:
```suggestion
<strong>Note: </strong> You can reuse the model you instantiated in the previous exercise.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Sorry, but I think the wording could be improved.

**Review by RandellDawson — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
Thank you, for your contribution, @Anonyma ! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**